kerrokantasi/models.py

```diff
@@ -26,11 +27,7 @@ class User(AbstractUser, SerializableMixin):
     # Properties for GDPR api serialization
     @property
     def sectioncomments(self):
-        return [s.serialize() for s in self.sectioncomment_created.everything().iterator()]
-
-    @property
-    def voted_sectioncomments(self):
-        return [s.serialize() for s in self.voted_democracy_sectioncomment.everything().iterator()]
+        return [s.serialize() for s in SectionComment.objects.everything().filter(created_by=self).iterator()]
```

Comment:
Not a huge thing, but I guess you can also do filtering in the everything method i.e.:
```suggestion
        return [s.serialize() for s in SectionComment.objects.everything(created_by=self).iterator()]
```

Reply:
fix'd